### PR TITLE
Show vesting data

### PR DESCRIPTION
**Pull request by aulneau**
### What this does

Test address page: https://explorer-7f6db7ey3.vercel.app/address/SP97B09CCCHE31FTHR97ZKAF0VF3DD1XR57MZSWX?chain=testnet

This PR pulls data for addresses that have vesting data and displays a card with some information:

Screen Shot 2021-01-09 at 1 43 55 PM

I've also added the ability to display when someone is viewing a mainnet/testnet address in the inverse context, and update the url by user action:

Screen Shot 2021-01-09 at 1 44 02 PM

### CODE REVIEW

File: src/common/hooks/use-network-mode.ts [hunk 1]
```diff
@@ -2,6 +2,6 @@ import { useRecoilValue } from 'recoil';
 import { networkModeState } from '@pages/_app';
 
 export const useNetworkMode = () => {
-  const state = useRecoilValue(networkModeState);
+  const state = useRecoilValue<'Testnet' | 'Mainnet' | null>(networkModeState);
```
Comment (kyranjamie) [inline src/common/hooks/use-network-mode.ts:5]: Do we want to have the capitalised value as the key in the store, versus lowercase key and formatting later to capitalise?

File: src/components/stacks-token-vesting-card.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { Box, Flex, color } from '@stacks/ui';
+import { Caption, Text } from '@components/typography';
+import { Section } from '@components/section';
+import { border, microToStacks } from '@common/utils';
+import { VestingData } from '@common/utils/addresses';
+
+export const StacksTokenVestingCard: React.FC<{ vesting: VestingData }> = ({ vesting }) => {
+  const percent = (vesting.totalUnlocked / vesting.vesting_total) * 100;
```
Comment (kyranjamie) [inline src/components/stacks-token-vesting-card.tsx:10]: unsafe math

**Review by kyranjamie — APPROVED** [review]

File: src/common/hooks/use-network-mode.ts (hunk 1 above)
Comment (aulneau) [reply src/common/hooks/use-network-mode.ts]: fixed, thank you

File: src/components/stacks-token-vesting-card.tsx (hunk 2 above)
Comment (aulneau) [reply src/components/stacks-token-vesting-card.tsx]: fixed!

**Pull request merged**